assignments/prototypes.js

```diff
@@ -22,6 +32,18 @@
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(attributes) {
```

Comment:
try to void naming your parameters the same thing, to avoid confusion. Maybe this one can be charAttrributes?